client/src/pages/Rides.js

```diff
@@ -24,6 +27,19 @@ const PageWrapper = styled.div`
 
 const Rides = () => {
   const history = useHistory();
+  useEffect(() => {
+    const doFetch = async () => {
+      const today = getCurrentDate();
+      const todaysRides = await getDataByQuery({
+        collectionName: "tasks",
+        dataName: "date",
+        query: today,
+      });
+      console.log(todaysRides);
```

Comment:
```suggestion

```

Reply:
Forgot to remove it. Thanks for reminding 🚀